packages/ui-signer/src/Extrinsic.tsx

```diff
@@ -40,7 +40,7 @@ class Transaction extends React.PureComponent<Props> {
           <div className='expanded'>
             <p>
               <Trans i18nKey='decoded.short'>
-                You are about to sign a message from <span className='code'>{accountId}</span> calling <span className='code'>{section}.{method}</span> with an index of <span className='code'>{accountNonce.toString()}</span>
+                You are about to sign a message from <span className='code'>{accountId}</span> calling <span className='code'>{section}.{method}</span> with an index of <span className='code'>{accountNonce ? accountNonce.toString() : null}</span>
```

Comment:
this one really is weird, I added the `?:` dance because calling toString on null was causing the app to crash which is strange because it should default to BN(0)...still investigating